Chapters 3/Chapters 3.pyproj

```diff
@@ -0,0 +1,37 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```

Comment:
please move these types of files to gitignore.

Reply:
I made changes to the commit :[b5e8de7]